Lib/compositor/cmap.py

```diff
@@ -4,7 +4,16 @@
 """
 
 def extractCMAP(ttFont):
-    return ttFont["cmap"].getcmap(3, 1).cmap
+    cmap = {}
+    cmapIDs = [(3, 10), (0, 3), (3, 1)]
+    for i in range(len(cmapIDs)):
```

Comment:
Thanks for the suggestions. In fact, we don't even need the `else`, we can just return the cmap as soon as it is found.